CONTRIBUTING.md

```diff
@@ -48,11 +48,13 @@ GitHub, clone, and develop on a branch. Steps:
    $ git push -u origin my-feature
    ```
 
+  *Note*: if your PR includes changes to files under the ``doc/`` or ``examples/`` folders, include in your commit message ``[doc build]``. Otherwise, include in your commit message ``[doc skip]``. This will allow the CI to avoid unnecessarily building the docs.
```

Comment:
I am not sure this is the best place to add this change. I feel it's better you add this as another bullet in the `Pull Request Checklist` section, below the part where documentation is mentioned.